validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
On Tue, May 05, 2015 at 07:57:46AM -0700, Mark Striemer wrote:
>I don't know what a "shared scope" is. Can you please explain that in the docstring?

I'll add a docstring, but essentially:

Most of these properties are flagged because when add-ons load 
code into a shared scope, such as a window, or web page, and 
replace or modify them, they cause conflicts with other code. 
This heuristic is the same as the previous version of the code, 
and assumes (incorrectly) that bootstrapped add-ons will never 
load code into a shared scope, and otherwise only recognizes 
JavaScript modules as private scopes.

The main change in this code is that rather than assume that 
`"readonly"` in general only applies to shared scopes, it now 
needs to be explicitly specified.